pyxem/utils/indexation_utils.py

```diff
@@ -90,9 +172,25 @@ def correlate_library(image, library, n_largest, mask):
     ----------
     E. F. Rauch and L. Dupuy, “Rapid Diffraction Patterns identification through
        template matching,” vol. 50, no. 1, pp. 87–99, 2005.
+
+    K. Briechle and U.D. Hanebeck, "Template Matching using Fast Normalized Cross Correlation"
```

Comment:
I think https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596837/ is a better reference.